src/index.jsx

```diff
@@ -34,12 +34,36 @@ const apiClient = getAuthenticatedAPIClient({
   loggingService: NewRelicLoggingService,
 });
 
+let callbackAccessToken = null;
+
+/**
+ * Temporary check of auth state vs callback access token to help debug and fix any
+ * potential problems.
+ */
+function checkAuthState(functionName, authenticationState) {
+  const userIdsMatch = !!(
+    callbackAccessToken && callbackAccessToken.user_id &&
+    authenticationState && authenticationState.authentication &&
+    authenticationState.authentication.userId &&
+    callbackAccessToken.user_id === authenticationState.authentication.userId
+  );
+  const customAttributes = {
+    functionName,
+    callbackAccessToken,
+    authenticationState,
+    userIdsMatch,
+  };
+  NewRelicLoggingService.logInfo(`checkAuthState: ${functionName}: userIdsMatch=${userIdsMatch}`, customAttributes);
```

Comment:
Are you outputting `functionName` twice?  Once in `customAttributes` and once after `checkAuthState:`.

Reply:
Yes. Same with `userIdsMatch`.  The former is the message which appears on the NewRelic JS Errors page.  The second is an object that can be used in NewRelic Insight's queries.